logfile/operations/types/convert_files.py

```diff
@@ -21,129 +24,71 @@ class ConvertFiles(OperationBase):
+    def _convert_file(filepath: str, new_extension: str) -> None:
         """
         Convert file to new extension and deletes the old file
 
         Args:
             filepath(str): Full filepath to file
             new_extension(str):  New extension to file example: ('.txt')
         """
-        path = Path(filepath)
-        path.rename(Path(path.parents[0], path.name + new_extension))
-
-    def __get_exclude_files_instruction(self):
-        """
-        Get file names to exclude from configuration instructions
-
-        Returns:
-            List<string>: Values to exclude
-            None
-        """
-        if self.__exclude_files_instruction_key in self.config:
-            value = self.config[self.__exclude_files_instruction_key]
-            return value.split('|')
-        return []
-
-    def __get_exclude_extensions_instruction(self):
-        """
-        Get file extensions to exclude from configuration instructions
-
-        Returns:
-            List<string>: Values to exclude
-            None
-        """
-        if self.__exclude_extensions_instruction_key in self.config:
-            value = self.config[self.__exclude_extensions_instruction_key]
-            return value.split('|')
-        return []
-
-    def __get_recursive_instruction(self):
-        """
-        Get recursive instruction from configuration instructions
-
-        Returns:
-            bool: Recursive behavior enabled
-        """
-        val = self.config[self.__recursive_instruction_key].lower()
-        return val == "true"
+        try:
+            path = Path(filepath)
+            new_path = Path(path.parents[0], path.name + new_extension)
+            path.rename(new_path)
+            logging.debug(f"Converted file '{path}' to '{new_extension}'")
+        except OSError as exc:
+            logging.warning(f"Could not convert file '{path}' to '{new_extension}' {exc}")
 
-    def __get_directory_instruction(self):
+    @staticmethod
+    def __list_item_contains_string(arr: list, item: str) -> bool:
         """
-        Get directory from configuration instructions
+        Checks if list items contains part of a string
 
-        Returns:
-            str: Directory
-        """
-        return self.config[self.__directory_instruction_key]
+        Example:
+            item = '1h'
+            arr = ['ge1h', 'ttt' , 'yyy']
+            returns True
 
-    def __get_new_file_extension_instruction(self):
-        """
-        Get new extension from configuration instructions
+        Args:
+            arr(list): List of string
+            item(str): Item to search for
 
         Returns:
-            str: New extension
-        """
-        return self.config[self.__new_file_extension_instruction_key]
-
-    @staticmethod
-    def __list_item_contains_string(arr, item):
-        """
+            bool: Item is found
         """
         for val in arr:
             if val in item:
```

Comment:
Shouldn't it be item in val?

Reply:
This is correct but the documentation was wrong. I fixed it

Reply:
OK, but then I think the name is misleading. What about `__list_item_in_string`